src/plugins/tables/reports/a11y-1-en.hbs

```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Accessibility assessment #1 - Tables",
+	"language": "en",
+	"description": "Partial. Evaluation of new Success Criteria at Level A and Level AA of WCAG 2.1 and WCAG 2.2",
```

Comment:
Make the same edit as proposed in the other pages

Reply:
done